File: backend-example-dotnet/src/application/test/TestIocClass1.cs
namespace BackendExample.Application.Test
{
    /// <summary>
    /// IoC测试类1
    /// </summary>
    public class TestIocClass1
    {
        private readonly TestIocClass2 class2;

        public TestIocClass1(TestIocClass2 class2)
        {
            this.class2 = class2;
        }
    }
}

File: backend-example-java/src/main/java/com/example/backend/application/test/TestIocClass1.java
package com.example.backend.application.test;

import org.springframework.stereotype.Component;

/**
 * IoC测试类1
 */
@Component
public class TestIocClass1 {

    private final TestIocClass2 service2;

    public TestIocClass1(TestIocClass2 service2) {
        this.service2 = service2;
    }
}
